fem/fe/fe_fixed_order.cpp

```diff
@@ -1809,15 +1809,12 @@ void RT0TriangleFiniteElement::Project (
 {
    double vk[2];
    Vector xk (vk, 2);
-#ifdef MFEM_THREAD_SAFE
-   DenseMatrix Jinv(dim);
-#endif
 
```

Comment:
I am not sure about all of these `Jinv` changes in `MFEM_THREAD_SAFE`... @v-dobrev, are they okay?

Reply:
I may be missing some subtlety but the thought was that we are calculating the transposed adjugate of the Jacobian which is itself cached in the element transformation object. If the cached Jacobian is "thread safe" then the new cached adjugate Jacobian will also be thread safe. If not, then we have a larger problem because we are trying to compute a thread safe object from something that is not thread safe.

Reply:
Ok, this sounds good. I'm just wary of thread safety -- it often requires local variables, that will likely shadow the ones on the class.